pkg/nsx/client.go

```diff
@@ -26,11 +26,16 @@ import (
 )
 
 const (
-	FeatureSecurityPolicy           string = "SECURITY_POLICY"
-	FeatureNSXServiceAccount        string = "NSX_SERVICE_ACCOUNT"
-	FeatureNSXServiceAccountRestore string = "NSX_SERVICE_ACCOUNT_RESTORE"
+	VPC = iota
```

Comment:
It's good to remove VPC and StaticRoute here, since we don't use it yet.

Reply:
also, we already remove VPC and StaticRoute version check in 
func (nsxVersion *NsxVersion) featureSupported(feature int) bool

Reply:
Since we may add new features here on both main and vpc_dev branch, we'd better to keep an union here to avoid many conflicts when cherry-pick changes to vpc_dev and to main.